### DISCUSSION TITLE
Installing firedrake - petsc installation error

### DISCUSSION THREAD

**Discussion by adofarsi** [firedrakeproject/firedrake#2799] [Firedrake support]
Hi! I'm trying to install Firedrake on Ubuntu, but the script returns an error when trying to install petsc.
Attached you can find both the Firedrake-install.log and the configure.log. Thank you for the help!
[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/10899942/firedrake-install.log)
[configure.log](https://github.com/firedrakeproject/firedrake/files/10899943/configure.log)

**Comment by connorjward** [comment]
So the relevant error is
```
/usr/bin/ld: uv-src-unix-loop.c.o: relocation R_X86_64_32 against symbol `uv__work_done' can not be used when making a PIE object; recompile with -fPIE
```

and it shows up when installing `cmake`. I don't know what could be causing this.

You may have some success if you comment [this line](https://github.com/firedrakeproject/firedrake/blob/master/scripts/firedrake-install#L719) out of the installation script (and replace it with `pass` so you don't get an indentation error). Make sure you've run `sudo apt install cmake` as well. As per the comment above make sure `cmake --version` is >= 3.18.1.
